src/features/pastelStatistics/components/chart/EChartsLineChart.tsx

```diff
@@ -0,0 +1,307 @@
+import * as echarts from 'echarts'
+import ReactECharts from 'echarts-for-react'
+import { saveAs } from 'file-saver'
+import * as htmlToImage from 'html-to-image'
+import React, { useEffect, useRef, useState } from 'react'
+import { CSVLink } from 'react-csv'
+import { Data } from 'react-csv/components/CommonPropTypes'
+import { TPeriod, makeDownloadFileName } from '../../utils/PastelStatisticsLib'
+
+import styles from './LineChart.module.css'
+
+type TThemeColor = {
```

Comment:
We are using the same code in the file `EChartsMultiLineChart.tsx`. Can we move them to the common folder?